chat-client/src/client/tabs/tabFactory.ts

```diff
@@ -84,23 +80,6 @@ export class TabFactory {
         return tabData
     }
 
-    private getWelcomeBlock() {
```

Comment:
IIRC these are also enabled on agentic chat branch in VSCode, was there a requirement to remove them?

Reply:
The description doesn't give any context why this change is made

Reply:
> IIRC these are also enabled on agentic chat branch in VSCode, was there a requirement to remove them?

yes, this is a task from the mile stone tracker. https://quip-amazon.com/7X13ASYyYxww/VSC-IDE-2025-Milestones#temp:s:temp:C:CQC4e7b1709330f4137a05c2bf3f;temp:C:CQCdec271714ac84255a8f310a08

Reply:
This seems to be in conflict with the figma though? Can you raise this on the tiger team channel? I'm confused which ones the source of truth

the code itself looks fine though

Reply:
discussed in team slack channel, this is a requirement from PM and the figma will be updated.